test/specs/scale.category.tests.js

```diff
@@ -49,6 +49,8 @@ describe('Category scale tests', function() {
 				labelOffset: 0,
 				minor: {},
 				major: {},
+				lineWidth: 0,
```

Comment:
why do we need these?

Reply:
well, this is set in the defaults, https://github.com/chartjs/Chart.js/pull/6787/files#diff-f0d8f86fecab28f396793911b0e9f36dR55 the tests expect to see them.

Reply:
Right, its that kind of test. Good.